pkg/sentry/platform/ring0/pagetables/walker_amd64.go

```diff
@@ -63,7 +60,7 @@ type Walker struct {
 // non-canonical ranges. If they do, a panic will result.
 //
 //go:nosplit
-func (w *Walker) iterateRange(start, end uintptr) {
+func (w *Walker) iterateRange(pt *PageTables, start, end uintptr) {
```

Comment:
Why all these changes? PageTables escapes by definition (https://github.com/google/gvisor/blob/master/pkg/sentry/platform/ring0/pagetables/pagetables.go#L45).

This change isn't too bad, but I don't think we want to make sweeping changes (especially if they hurt readability) just because this tool has unhelpful results.